README.md

```diff
@@ -2,24 +2,24 @@
-| [![Vagas.com](/companies/vagas.png?raw=true)](https://www.vagas.com.br)                          | [vagas.com.br/trabalhe-conosco](https://www.vagas.com.br/trabalhe-conosco)       |       | São Paulo-SP                                                                                                                                                             |         |
+| Empresa                                                                                          | Lista de vagas                                                                   | Devs  | Cidade                                                                                                                                                                                         | Remoto? |
+| :---                                                                                             | :---                                                                             | :---: | :---                                                                                                                                                                                           | :---:   |
+| [![App Prova](/companies/appprova.png?raw=true)](http://appprova.com.br/)                        | [https://appprova.recruiterbox.com/](https://appprova.recruiterbox.com/)         |       | Belo Horizonte-MG                                                                                                                                                                              |         |
+| [![Bio Ritmo](/companies/bio-ritmo.png?raw=true)](https://www.bioritmo.com.br)                   | [bioritmo.com.br/trabalhe_conosco](https://www.bioritmo.com.br/trabalhe_conosco) |       | São Paulo-SP                                                                                                                                                                                   |         |
+| [![CloudWalk](/companies/cloud-walk.png?raw=true)](https://www.cloudwalk.io/pt)                  | [cloudwalk.io/pt/careers](https://www.cloudwalk.io/pt/careers)                   |       | São Paulo-SP                                                                                                                                                                                   |         |
+| [![Codeminer 42](/companies/codeminer.png?raw=true)](https://www.codeminer42.com)                | [become@codeminer42.com](mailto:become@codeminer42.com)                          | 75    | São Paulo-SP<br>Campinas-SP<br>Novo Hamburgo-RS<br>Natal-RN<br>Sorocaba-SP<br>Teresina-PI<br>Anápolis-GO<br>Goiânia-GO<br>Batatais-SP<br>Guarapuava-PR<br>Santa Maria-RS<br>Poços de Caldas-MG | ✕       |
```

Comment:
Thanks, @Kutomore!

`Poços de Caldas-MG` isn't in [Codeminer's website](https://www.codeminer42.com/) though 😅

Reply:
We are working on this, will be there soon but it's already open and working! <3

Reply:
Awesome! 😃